src/storage/query/QueryVertexPropsProcessor.cpp

```diff
@@ -88,6 +88,9 @@ kvstore::ResultCode QueryVertexPropsProcessor::collectVertexProps(
             // Already found the latest version.
             continue;
         }
+        // Build TTL info
+        buildTagTTLInfo(tagId);
```

Comment:
Could you combine it with getTagTTLInfo.  We don't need to lookup the hash map twice.

Reply:
Good catch，fixed it.
Please review it again.